### DISCUSSION TITLE
Evaluate using memfd_secret() for Linux >= 5.14

### DISCUSSION THREAD

**Discussion by cypherbits** [jedisct1/libsodium#1134] [Ideas]
Linux 5.14 has the new memfd_secret syscall, it would be good to use it. 
More info: https://lwn.net/Articles/865256/

**Comment by jedisct1** [comment]
Can you clarify when, how and why that system call would be useful?

Now that the uncached mappings flag has been removed, it doesn't seem to be very useful any more.
